src/pages/HomePage/DummyData/DummyData.type.ts

```diff
@@ -0,0 +1,12 @@
+export type DummyLanguageType = 'python' | 'javascript' | 'cPlusPlus' | 'java';
+
+export interface RoomDataProps {
+  id: number;
+  title: string;
+  roomAccess: boolean;
+  currentRoomMember: number;
+  roomMemberLimit: number;
+  tags: string[];
+  language: DummyLanguageType[];
```

Comment:
p5;
저희가 기존에도 4개의 언어 외에는 고려안했던 것 같아서 타입명을 `LanguageType`로 해도 될 거 같아요!
```suggestion
  language: LanguageType[];
```

Reply:
넵! 해당 부분은 추후 api 현동하며 수정하도록 하겠습니다~